demosplan/DemosPlanCoreBundle/Command/Data/GenerateStatementCommand.php

```diff
@@ -10,7 +10,7 @@
 
 namespace demosplan\DemosPlanCoreBundle\Command\Data;
 
-use demosplan\DemosPlanCoreBundle\DataGenerator\Factory\StatementFactory;
+use demosplan\DemosPlanCoreBundle\DataGenerator\CustomFactory\StatementFactory;
```

Comment:
I don't fully understand why you changed the namespaces from factory to CustomFactory. Could you explain that please?

Reply:
Ok, makes sense to me. But it also sounds as if we should check if maybe these custom factories could be replaced with foundry at some point. Just as an idea, not relevant now.

Reply:
Yes, I thinks quite likely.